src/ios/IndoorAtlasWayfinding/IAWayfinding.h

```diff
@@ -0,0 +1,26 @@
+//
+//  IAWayfinding.h
+//
+//  Created by Toni on 27/11/2017.
+//
+
+#ifndef IAWayfinding_h
```

Comment:
Same thing for iOS. no need for a separate Cordova wayfinding plugin: merge these to the main plugin